content/docs/concepts/performance-and-scalability/index.md

```diff
@@ -13,127 +13,120 @@ aliases:
+Since the proxy normally doesn't buffer the data passing through,
+request rate doesn't affect the memory consumption.
 
-Here is an example of scenario (one out of the 8 scenarios we run for every build) result graphing the latency distribution for `istio-0.7.1` at 400 Query-Per-Second (qps) between 2 services inside the mesh (with mutual TLS, Mixer policy checks and telemetry collection):
+### Latency
 
-<iframe src="https://fortio.istio.io/browse?url=qps_400-s1_to_s2-0.7.1-2018-04-05-22-06.json&xMax=105&yLog=true" width="100%" height="1024" scrolling="no" frameborder="0"></iframe>
+Since Istio injects a sidecar proxy on the data path, latency is an important
+consideration. Istio adds an authentication and a Mixer filter to the proxy. Every
+additional filter adds to the path length inside the proxy and affects latency.
 
-Comparing 0.6.0 and 0.7.1 histograms/response time distribution for the same scenario, clearly showing 0.7 improvements:
+The Envoy proxy collects raw telemetry data after a response is sent to the
+client. The time spent collecting raw telemetry for a request does not contribute
+to the total time taken to complete that request. However, since the worker
+is busy handling the request, the worker won't start handling the next request
+immediately. This process adds to the queue wait time of the next request and affects
+average and tail latencies. The actual tail latency depends on the traffic pattern.
 
-<iframe src="https://fortio.istio.io/?xMin=2&xMax=110&xLog=true&sel=qps_400-s1_to_s2-0.7.1-2018-04-05-22-06&sel=qps_400-s1_to_s2-0.6.0-2018-04-05-22-33" width="100%" height="1024" scrolling="no" frameborder="0"></iframe>
+Inside the mesh, a request traverses the client-side proxy and then the server-side
+proxy. This two proxies on the data path add about `10ms` to the 99th percentile latency at `1k rps`.
+The server-side proxy alone adds `6ms` to the 99th percentile latency.
 
-And tracking the progress across all the tested releases for that scenario:
+### Latency for Istio {{< istio_release_name >}}
 
-<iframe src="https://fortio.istio.io/?s=qps_400-s1_to_s2" width="100%" height="1024" scrolling="no" frameborder="0"></iframe>
+The default configuration of Istio 1.1 adds `10 ms` to the 99th percentile latency of the data plane over the baseline.
+We obtained these results using the [Istio benchmarks](https://github.com/istio/tools/tree/master/perf/benchmark)
+for the `http/1.1` protocol, with a `1 kB` payload at `1000 rps` using 16 client connections, 2 proxy workers and `mTLS` enabled.
 
-You can learn more about [Fortio](https://github.com/fortio/fortio/blob/master/README.md#fortio) on GitHub and see results on [https://fortio.istio.io](https://fortio.istio.io).
+{{< image width="90%" ratio="75%"
+    link="latency.svg?sanitize=true"
+    alt="P99 latency vs client connections"
```

Comment:
Same comment as before about 90%ile vs 99%ile - should we be reporting at 90%ile?